tests/experimental/datasources/test_integration_datasource.py

```diff
@@ -257,13 +265,8 @@ def test_run_data_assistant_and_checkpoint(datasource_test_data):
 def multibatch_sql_data(
     context: AbstractDataContext,
 ) -> tuple[AbstractDataContext, Datasource, DataAsset, BatchRequest]:
-    db_file = (
-        pathlib.Path(__file__)
-        / "../../../test_sets/taxi_yellow_tripdata_samples/sqlite/yellow_tripdata_sample_2020_all_months_combined.db"
-    ).resolve()
-    datasource = context.sources.add_sqlite(
-        name="test_datasource",
-        connection_string=f"sqlite:///{db_file}",
+    datasource = sqlite_datasource(
```

Comment:
@billdirks In #6981 `multibatch_sql_data` is moved to a util module (the merge should not be too difficult).